README.md

```diff
@@ -78,7 +78,22 @@ Follow this guide to setup your development machine.
     ```
     python scripts/workers/submission_worker.py
     ```
+#### Using Docker
```

Comment:
Sure thing. I assumed a sub-subheading would have been appropriate. Thanks for the clarification. :)